src/Chapter1.hs

```diff
@@ -489,8 +492,10 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit n = mod num 10
+  where
+    num = if n >= 0 then n else negate n
```

Comment:
Nice and correct implementation! Though, you can use a standard function `abs` here that already does a similar thing :slightly_smiling_face: